pom.xml

```diff
@@ -1146,6 +1149,38 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>net.revelc.code.formatter</groupId>
+        <artifactId>formatter-maven-plugin</artifactId>
+        <version>${formatter-maven-plugin.version}</version>
```

Comment:
Remove spotless?

Reply:
Definitely!
But, since those plugins are not activated by default(for now) I plan to remove spotless and apply the new format in the entire `java-generator` module in an atomic follow up PR.